Cirrious.FluentLayout/FluentLayoutExtensions.cs

```diff
@@ -18,8 +18,8 @@ public static void SubviewsDoNotTranslateAutoresizingMaskIntoConstraints(this UI
             foreach (var subview in view.Subviews)
             {
                 subview.TranslatesAutoresizingMaskIntoConstraints = false;
-		
-	        if (subview.Subviews != null && subview.Subviews.Count() > 0)
+
+                if ((subview as UIControl) == null && subview.Subviews != null && subview.Subviews.Count() > 0)
```

Comment:
However, that means if I create my own view that subclasses `UIView` and nest other `UIView` inside, this piece of code won't work.